plugins/modules/custom_image.py

```diff
@@ -178,22 +172,101 @@
 from ansible.module_utils.basic import (
     AnsibleModule,
 )
+from ansible.module_utils.urls import (
+    fetch_url
+)
 from ..module_utils.api import (
+    AnsibleCloudscaleApi,
     AnsibleCloudscaleBase,
     cloudscale_argument_spec,
 )
+from ansible.module_utils._text import to_text
+import re
+import json
+import time
+
+
+class AnsibleCloudscaleCustomImage(AnsibleCloudscaleBase):
+
+    def fetch_both_urls(self, api_call):
+        headers = self._auth_header
+
+        # Get image information
+        url = '%s%s' % (self._api_url, api_call)
+        response, info = fetch_url(self._module,
+                                   url,
+                                   headers=headers,
+                                   timeout=self._module.params['api_timeout'])
+
+        # Add import to the url
+        if api_call == 'custom-images':
+            api_call = 'custom-images/import'
+        else:
+            api_call = re.sub(r"/", "/import/", api_call)
```

Comment:
Why do you use a regex for this? Wouldn't it be simpler to just append `import`?